src/components/search/GlobalSearchField.js

```diff
@@ -325,6 +334,50 @@ function AnalysesSearchOption(props) {
     );
 }
 
+function TeamSearchOption(props) {
+    const theme = useTheme();
+    const { t: i18NSearch } = useTranslation("search");
+    const { baseId, filter, selectedOption, searchTerm } = props;
+
+    const id = selectedOption?.id;
+
+    if (
+        id === searchConstants.VIEW_ALL_ANALYSES_ID ||
+        id === searchConstants.VIEW_ALL_APPS_ID ||
+        id === searchConstants.VIEW_ALL_DATA_ID ||
+        id === searchConstants.VIEW_ALL_TEAMS_ID
+    ) {
+        const [prompt, selectedTab] = getViewAllPrompt(
+            id,
+            searchTerm,
+            i18NSearch
+        );
+        return (
+            <ViewAllOption
+                searchTerm={searchTerm}
+                filter={filter}
+                prompt={prompt}
+                selectedTab={selectedTab}
+                id={build(baseId, ids.VIEW_ALL)}
+            />
+        );
+    }
+
+    const href = `/${NavigationConstants.TEAMS}`;
+    const as = `/${NavigationConstants.TEAMS}`;
+    return (
+        <Link href={href} as={as} passHref>
+            <SearchOption
+                primary={selectedOption.display_extension}
+                secondary={selectedOption.description}
+                icon={<TeamIcon color={theme.palette.info.primary} />}
```

Comment:
I don't think this works. You will have to use style instead.
`style={{theme.palette.info.main}}`